app/src/main/java/com/duckduckgo/app/browser/webview/MaliciousSiteBlockerWebViewIntegration.kt

```diff
@@ -54,6 +55,13 @@ interface MaliciousSiteBlockerWebViewIntegration {
     ): Boolean
 
     fun onPageLoadStarted()
+
+    fun onSiteExempted(url: Uri)
+}
+
+@SingleInstanceIn(AppScope::class)
+class ExemptedUrlsHolder @Inject constructor() {
+    val exemptedMaliciousUrls = mutableSetOf<String>()
```

Comment:
we shouldn't expose a mutable set, but rather provide a method for exemptions to be added